packages/react-components/src/organisms/ManuscriptVersionCard.tsx

```diff
@@ -401,10 +523,10 @@ const ManuscriptVersionCard: React.FC<ManuscriptVersionCardProps> = ({
               )}
               <Caption accent="lead" noMargin>
                 <div css={userContainerStyles}>
-                  Date added:
+                  Date created:
```

Comment:
Just confirming that there was a design change.

Reply:
I'm not sure if this was changed across the whole figma, but we discussed this in this thread https://yld.slack.com/archives/CV4UY4411/p1730721284620429?thread_ts=1730460907.784259&cid=CV4UY4411 :)

Reply:
Thanks, confirmed